### PR TITLE
Bah 1161 - Internationalization UAT bug fixes and label changes

### PR DESCRIPTION
**Pull request by buvaneswari-arun**

### CODE REVIEW

File: ui/app/adt/controllers/adtController.js [hunk 1]
```diff
@@ -29,7 +29,23 @@ angular.module('bahmni.adt')
...
 
+                    var keyName = attribute.toUpperCase().replace(/\s\s+/g, ' ').replace(/[^a-zA-Z0-9 _]/g, "").trim().replace(/ /g, "_");
+                    var translationKey = keyPrefix + "_" + keyName;
+                    var translation = $translate.instant(translationKey);
+                    if (translation != translationKey) {
+                        attribute = translation;
+                    }
+                }
+                return attribute;
+            };
```
Comment (binduak) [inline ui/app/adt/controllers/adtController.js:48]: @buvaneswari-arun we already have this method in the translationUtil.js. Then why are we not using it here ?

Comment (buvaneswari-arun) [reply ui/app/adt/controllers/adtController.js:48]: TranslationUtils.translateAttribute method is not accessible from this bedmanagement. Hence forced to put the content of the method directly here.

Comment (binduak) [reply ui/app/adt/controllers/adtController.js:48]: @buvaneswari-arun is it the same case with other modules ?

Comment (buvaneswari-arun) [reply ui/app/adt/controllers/adtController.js:48]: I tried with Bedmanagement - as it was part of the. card - not sure about IPD and programs etc and they will be played in the upcoming releases

**Review by binduak — APPROVED** [review]

**Pull request merged**